vocode/streaming/synthesizer/eleven_labs_synthesizer.py

```diff
@@ -67,8 +69,29 @@ async def create_speech(
 
         cache_key = self.get_cache_key(message.text)
         audio_data = self.cache.get(cache_key)
-        if audio_data is None:
-            self.logger.debug("Synthesizing message - message not found in cache")
+
+        start_time = datetime.datetime.utcnow()
+        if audio_data is not None:
+            tts_log = TTSLog(
+                conversation_id="",
```

Comment:
will these logs be lost to the abyss if we dont provide a `conversation_id` here and below?

Reply:
They will be dumped into file (file handler) (create_speech() does not have conversation id. I will add as optional argument so we can pass to log in 11labs / azure tts

Reply:
done